lib/chef/application/exit_code.rb

```diff
@@ -38,6 +38,8 @@ class ExitCode
         # 42 was used by audit mode and should not be reused
         CONFIG_FAILURE: 43,
         CLIENT_UPGRADED: 213,
+        LICENSE_NOT_ENTITLED: 173,
+        LICENSE_NOT_SET: 174,
```

Comment:
lets keep these in order, please.

Reply:
This is not needed in this PR and removed it.